test/unit/base.test.js

```diff
@@ -292,6 +293,50 @@ describe('CONVERT FUNCTION TESTS ', function() {
         done();
       });
     });
+    it('convertor should maintain implicit headers in the request' +
+    implicitHeadersSpec, function(done) {
+      var openapi = fs.readFileSync(implicitHeadersSpec, 'utf8');
+      Converter.convert({ type: 'string', data: openapi }, {
+        schemaFaker: true,
+        keepImplicitHeaders: true
+      }, (err, conversionResult) => {
+        expect(err).to.be.null;
+        expect(conversionResult.output[0].data.item[0].item[0].request.header[0].key)
+          .to.equal('Authorization');
+        expect(conversionResult.output[0].data.item[0].item[0].request.header[0].value)
+          .to.equal('Bearer {{oauth_access_token}}');
+        expect(conversionResult.output[0].data.item[0].item[0].request.header[1].key)
+          .to.equal('Content-Type');
+        expect(conversionResult.output[0].data.item[0].item[0].request.header[1].value)
+          .to.equal('application/json');
+        expect(conversionResult.output[0].data.item[0].item[1].request.header[0].key)
+          .to.equal('Authorization');
+        expect(conversionResult.output[0].data.item[0].item[1].request.header[0].value)
+          .to.equal('Bearer {{oauth_access_token}}');
+        expect(conversionResult.output[0].data.item[0].item[1].request.header[1].key)
+          .to.equal('Content-Type');
+        expect(conversionResult.output[0].data.item[0].item[1].request.header[1].value)
+          .to.equal('application/json');
+        expect(conversionResult.output[0].data.item[0].item[1].request.header[2]).to.equal(undefined);
```

Comment:
Following would be more appropriate as mentioned above.
```
expect(conversionResult.output[0].data.item[0].item[1].request.header).to.have.length(2);
```

Reply:
I corrected the test as suggested